variables.tf

```
@@ -4,36 +4,98 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)"
+variable "auditors" {
+  type = map
```

Should this be:

```suggestion
  type = map(string)
```
?

---

`map(string)` won't work here because `auditors` is a map whose keys are strings, but whose values are maps.  Terraform expects a `map(string)` variable to have strings for **both** the keys and values.

---

I believe `map(map(list(string)))` would be the correct type here.  A map with values that are maps with values that are lists of strings.

---

@mcdonnnj and I were also discussing this on a side chat.  I will change this to `map(map(list(string)))` in an upcoming PR.

---

This has been rectified in https://github.com/cisagov/cool-auditor-iam/pull/3.  Thanks all!